locator/token_metadata.cc

```diff
@@ -913,15 +913,16 @@ future<> token_metadata_impl::update_pending_ranges(
 }
 
 size_t token_metadata_impl::count_normal_token_owners() const {
-    return _nr_normal_token_owners;
+    return _normal_token_owners.size();
 }
 
-void token_metadata_impl::update_normal_token_owners() {
+future<> token_metadata_impl::update_normal_token_owners() {
     std::unordered_set<inet_address> eps;
     for (auto [t, ep]: _token_to_endpoint_map) {
         eps.insert(ep);
+        co_await coroutine::maybe_yield();
```

Comment:
This is going to make backport harder since we do not have coroutine::maybe_yield in 2021.1.

Reply:
2022.1 would be an easy target for backporting
but I'm not sure if the issue is severe enough to warrant backporting to 2021.1
We can deal with it if and when this is backported there.

See discussion in https://github.com/scylladb/scylla-enterprise/issues/2194#issuecomment-1202046266
There are quite a few optimizations around token_metadata (and particularly effective_replication_map) that aren't feasible for backporting there.

Reply:
OK.